pkg/pipelines/pipelines.go

```diff
@@ -344,6 +344,30 @@ func ToPipelineActivity(pr *v1beta1.PipelineRun, pa *v1.PipelineActivity, overwr
 	}
 
 	activities.UpdateStatus(pa, false, nil)
+
+	for k := range pr.Status.Conditions {
+		msg := pr.Status.Conditions[k].GetMessage()
+
+		if strings.Contains(msg, "PipelineRun") {
+			msg = strings.ReplaceAll(msg, "PipelineRun", "PipelineActivity")
+		}
+
+		if strings.Contains(msg, pa.Labels["tekton.dev/pipeline"]) {
+			msg = strings.ReplaceAll(msg, pa.Labels["tekton.dev/pipeline"], pa.Name)
+		}
+
+		pa.Spec.Message = v1.ActivityMessageType(msg)
+	}
+
+	k1 := 0
```

Comment:
Why k1? and k2? k is scoped to only the for loop, and you should be able to reuse k.

Reply:
k is for the first loop,
k1 is for the pa count
k2 is for the second loop